Use aria-hidden when hiding sidebar

brockfanning opened the pull request:
Fixes #931 

This doesn't use `display: none` because I believe that would mess up our animation. Instead this uses `aria-hidden` which should give the same result.

LucyGwilliamAdmin commented:
@brockfanning I've just tried this out but it still seems to be there in the background e.g. if you go [here](http://sdgdev-813006012.eu-west-1.elb.amazonaws.com/aria-hidden-sidebar/3-a-1/) and tab through, when you get to goal icon you will have to tab a few times to get to the next visible focusable item

brockfanning commented:
@LucyGwilliamAdmin Good catch. I guess we're going to have to give up on our animation here. I've just removed the animation and switched to `display:none`.

LucyGwilliamAdmin reviewed (approved):
Sad that we've lost the animation but it does work now:
http://sdgdev-813006012.eu-west-1.elb.amazonaws.com/aria-hidden-sidebar/3-a-1/
